source/code/plugins/scom_configuration.rb

```diff
@@ -59,11 +61,16 @@ def self.load_scom_configuration
         OMS::Log.error_once("Failed to read certificate or key from #{@@cert_path} #{@@key_path}")
         return false
       end # begin
+       
+      @@fqdn = OMS::Common.get_fully_qualified_domain_name
+      if(!@@fqdn)
+        OMS::Log.error_once("Unable to get FQDN")
```

Comment:
get_fully_qualified_domain_name already logs this error, so you shouldn't log it a second time.